src/integrationTest/java/org/opensearch/security/privileges/ActionPrivilegesTest.java

```diff
@@ -258,6 +262,69 @@ public void hasAny_wildcard() throws Exception {
                 isForbidden(missingPrivileges("cluster:whatever"))
             );
         }
+
+        @Test
+        public void apiToken_explicit_failsWithWildcard() throws Exception {
+            SecurityDynamicConfiguration<RoleV7> roles = SecurityDynamicConfiguration.fromYaml("test_role:\n" + //
```

Comment:
If the roles in this are unused, can we use `SecurityDynamicConfiguration.empty(CType.ROLES);` instead?

Reply:
Applied the suggested change to all instances where I did not need to use the roles.